test/functional/ui/screen_basic_spec.lua

```diff
@@ -73,33 +73,29 @@ describe('Screen', function()
   describe(':suspend', function()
     it('is forwarded to the UI', function()
       local function check()
-        if not screen.suspended then
-          return 'Screen was not suspended'
-        end
+        eq(true, screen.suspended)
       end
```

Comment:
Does this fail correctly? IIRC screen:expect() does not catch test failures, only errors.